pkg/terminal/console_windows.go

```diff
@@ -0,0 +1,35 @@
+// +build windows
+
+package terminal
+
+import (
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/windows"
+)
+
+// SetConsole switches the windows terminal mode to be able to handle colors, etc
+func SetConsole() error {
+	if err := setConsoleMode(windows.Stdout, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING); err != nil {
+		return err
+	}
+	if err := setConsoleMode(windows.Stderr, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING); err != nil {
+		return err
+	}
+	if err := setConsoleMode(windows.Stdin, windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING); err != nil {
+		return err
+	}
+	return nil
+}
+
+func setConsoleMode(handle windows.Handle, flags uint32) error {
+	var mode uint32
+	err := windows.GetConsoleMode(handle, &mode)
+	if err != nil {
+		return err
+	}
+	if err := windows.SetConsoleMode(handle, mode|flags); err != nil {
+		logrus.WithError(err).Error("Failed to set console mode for cli")
```

Comment:
Should this return the error, if not could you add a comment as to why it should not?